modules/swagger-core/src/test/java/io/swagger/jackson/XMLInfoTest.java

```diff
@@ -34,14 +35,19 @@ public void testSimple() throws Exception {
         assertNotNull(xml);
         assertEquals(xml.getName(), "xmlDecoratedBean");
 
-        final Property property = impl.getProperties().get("items");
+        // Cast it to an array property
+        final ArrayProperty property = (ArrayProperty)impl.getProperties().get("items");
         assertNotNull(property);
         final Xml propertyXml = property.getXml();
-
         assertNotNull(propertyXml);
-        assertEquals(propertyXml.getName(), "item");
+        // Check the name of array property
+        assertEquals(propertyXml.getName(), "items");
         assertTrue(propertyXml.getWrapped());
-
+        // Get the xml for items for the array property
+        final Xml itemsXml = property.getItems().getXml();
```

Comment:
@webron is this correct?  That the _wrapped_ value would take the name `items` and the XML property itself would be `item`?  I believe it's the opposite, but I've been staying far away from XML-land.